peglib.h

```diff
@@ -3400,6 +3433,53 @@ class parser
     std::string              start_;
 };
 
+
+/*-----------------------------------------------------------------------------
```

Comment:
I want to separate actual output trace format from peglib itself. It's an application's responsibly, because each application may want different style of trace output. Also some applications don't want to output messages to `std::cout`, but other devices such as std::cerr or a file.